asab/library/service.py

```diff
@@ -366,3 +366,14 @@ async def export(self, path="/", tenant=None):
 		tarobj.close()
 		fileobj.seek(0)
 		return fileobj
+
+	def subscribe(self, paths):
```

Comment:
Subscribing list of paths - it makes sense to me when most of the subscriptions happen during init time and may require more than one path